source/concepts.tex

```diff
@@ -478,13 +482,13 @@
 \begin{itemdescr}
 \pnum
 \begin{note}
-\tcode{\libconcept{SignedIntegral}<T>} can be satisfied even by types that are
+\tcode{\libconcept{SignedIntegral}<T>} can be modeled even by types that are
```

Comment:
Should the `<T>` be here if we're talking about modelling? Other unary concepts seem to be used without, e.g. `X` models `Y` (rather than `Y<X>`), and we only have a template argument list for binary concepts, e.g. `X` and `Y` model `Z<X,Y>`. Or have I misunderstood the convention?

Reply:
I agree the minimal change is too minimal here (and on 491). I'll remove the `<T>`.